### PR TITLE
create @types/customize-cra

### PR DESCRIPTION
**Pull request by k-yle**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## customize-cra ([<kbd>unpkg</kbd>](https://unpkg.com/browse/customize-cra@latest/))
was missing the following properties:
1. addWebpackPlugin
1. addWebpackResolve
1. getBabelLoader
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 06c6b4c974720e915664b9433da9a08a5528a2bc
</p>

File: types/customize-cra/src/core.d.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ *   condition && disableEsLint(),
+ * );
+ */
+export function override(...args: Array<OverrideFunc | false | undefined | null | 0>): OverrideFunc;
+
+/**
+ * Enables left to right composition of multiple override functions,
+ * only for dev mode.
+ */
+export function overrideDevServer(...args: OverrideFunc[]): OverrideFunc;
```
Comment (amcasey) [inline types/customize-cra/src/core.d.ts:27]: This one doesn't allow `undefined`, etc?

**Review by amcasey — APPROVED** [review]
One question

File: types/customize-cra/src/core.d.ts (hunk 1 above)
Comment (k-yle) [reply types/customize-cra/src/core.d.ts:27]: it's not in the documentation, so I suspect no...

**Pull request merged**